src/Ledger/PParams.lagda

```diff
@@ -14,6 +14,11 @@ open import MyDebugOptions
 open import Ledger.Prelude
 open EpochStructure es
 \end{code}
+
+Adjustable parameters of the Cardano ledger. They are used in block
```

Comment:
Should we use another font, like italic or bold, for phrases or headings that are not complete sentences? (e.g., _Adjustable parameters of the Cardano ledger_.)

Reply:
Yeah, there's lots of further improvements that we can do here. I just wanted to get started with something.